app/models/Scholarship.php

```diff
@@ -50,7 +52,7 @@ public static function isClosed($type = 'application')
           $end_date = self::getCurrentScholarship()->nomination_end;
       }
 
-      return date_has_expired($end_date);
+      return date_has_expired($end_date) || !date_has_expired($start_date);
```

Comment:
This feels a little strange to me (the "has not expired" phrasing), but it's probably not worth making another helper or doing too much refactoring for such a small bug fix. One thing I've found really useful elsewhere is the Carbon [comparison methods](http://carbon.nesbot.com/docs/#api-comparison) combined with Eloquent's date casts, so you could just do:

```php
$today = Carbon::now();

return $end_date->lt($today) || $start_date->gt($today);
```

Reply:
Yeah it seems strange to me too but I definitely agree with you about not refactoring this too much just for this fix.

Do you think using Carbon should be included in this fix?

Reply:
Wasn't sure if both parts were for later

Reply:
Naaaaah, I think this is good to merge as is!